### PR TITLE
Homepage partner's section #3

### PR DESCRIPTION
**Pull request by Makig10**
# Description
Created the homepage partner's section and styled it as intended. This is linked to issue #3.
Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+.logo{
+    margin-right: 33%;
+    width:153px;
+    height:65px;
+}
+.homeLinks {
+    /*outline:3px solid black;*/
+    justify-content: space-between;
+    margin-top:12px ;
+    margin-right: 20%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use a single margin property for top and right, i.e margin 10 10 10 10;

**Review by JAPHETHNYARANGA — COMMENTED** [review]
please resolve the conflicts , then share screen shot

**Comment by Makig10** [comment]
Noted.

File: src/css/style.css (hunk 1 above)
Comment (Makig10) [reply src/css/style.css]: Aright. Thank you.

**Comment by Makig10** [comment]
Screenshot from 2023-10-03 20-07-46
Screenshot from 2023-10-03 20-07-05

**Comment by Makig10** [comment]
Screenshot from 2023-10-03 20-21-08
  

Screenshot from 2023-10-03 20-21-56

**Comment by JAPHETHNYARANGA** [comment]
Good work, but closing this pr as issue is done, please pick another issue

**Pull request closed without merging**